dotCMS/dependencies.gradle

```diff
@@ -1,6 +1,7 @@
 //noinspection GroovyAssignabilityCheck
 dependencies {
 
+    compile group: 'org.apache.ant', name: 'ant', version: '1.9.7'
```

Comment:
We already have https://commons.apache.org/proper/commons-compress/apidocs/org/apache/commons/compress/archivers/tar/TarArchiveInputStream.html available in the ticka jar. Would that work as well? It seems like it might.